### ISSUE TITLE
api route does not work after 9.1.7

### ISSUE THREAD

**Issue by kle97** [rapi-doc/RapiDoc#734]
I'm using rapidoc version 9.1.3 with angular and everything works fine.
Rapidoc will scroll to hash location of api doc according to the url on location bar, for example "#post-/api/auth/login", but after updating to 9.1.7, this behavior does not work anymore. I suspect there was a change in route-prefix in the 9.1.7 release that changed the behavior.

**Comment by mrin9** [comment]
does it work with latest release `9.2.0` ?

Is there a reason you want to stick to `9.1.7` ?

**Comment by kle97** [comment]
It tried the latest version and it also didn't work. I think I had 2 high vulnerabilities from rapidoc dependencies that was resolved only from 9.1.7 onwards.

**Comment by kle97** [comment]
I want to clarify the problem that the scrolling part still works in read mode or focused mode. I'm using focused mode, and when I reload the page with the url hash location, rapidoc return to overview location instead of the desired location. Goto path doesn't work either. I checked your example and everything works fine, but in my case, I use rapidoc with angular. I imported rapidoc into angular component and everything seem worked fine in previous release.
